app/controllers/posts_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
   def show
     @post = Post.find(params[:id])
     @user = User.find(@post.author_id)
+    @comments = @post.comments.includes(:author)
```

Comment:
- [ ] As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=fetching%20all%20posts%20and%20their%20comments%20for%20a%20user) requirement, please make sure to fix the N+1 problem when fetching all posts and their comments for a user which can be done using the following:

```ruby
@posts = @user.posts.includes(:comments)
```

Reply:
Thank you for the tip, @ShadyShawkat ! I'll do the change right now!